.rhcicd/pr_check.sh

```diff
@@ -36,3 +36,13 @@ cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
     <testcase classname="dummy" name="dummytest"/>
 </testsuite>
 EOF
+
+# adding floorist tests.
+export COMPONENT_NAME="policies-engine"
+# I also need to pass in FLOORPLAN_2_TEST.
+export IQE_ENV_VARS="FLOORPLAN_TO_TEST=policies-backend-hms"
+export IQE_PLUGINS="floorist"
+export IQE_MARKER_EXPRESSION="floorist_smoke"
+export IMAGE="quay.io/cloudservices/iqe-tests:floorist"
```

Comment:
Don't override `IMAGE`. That env var sets the service that runs, not a test image. The service is policies-ui-backend.

Reply:
This controls what image is used for the pod running iqe as I understand. The pod will not deploy using policies-ui-backend.

Reply:
you want to use `IQE_IMAGE_TAG`

Reply:
https://github.com/RedHatInsights/cicd-tools/blob/main/cji_smoke_test.sh#L8

Reply:
I replaced IMAGE with IQE_IMAGE_TAG.

Reply:
ack - it should just be the tag though, not the whole image URI. 

```  -i, --image-tag TEXT          image tag to use for IQE pod```

Reply:
Updating...